content/docs/0.14.x/reference/bridge/manage_sequences/index.md

```diff
@@ -5,6 +5,90 @@ weight: 40
 keywords: [0.14.x-bridge]
 ---
 
+## Trigger new sequences
+
+To trigger new sequences, head to the environment view. There you can find the button "Trigger a new sequence". This button can also be found on the sequences view, and will redirect you to the the environment view too.
+
+{{< popup_image
+link="./assets/trigger-01-environment-screen.png"
+caption="Sequences">}}
+
+{{< popup_image
+link="./assets/trigger-02-sequence-screen.png"
+caption="Sequences">}}
+
+The first step in triggering a sequence is to select the sequence, as well as the service and stage you want to trigger the sequence for. For sequences you have the possibility to choose from 3 sequence types:
```

Comment:
I would start with this since the first thing top-bottom is the sequence type
```suggestion
For sequences you have the possibility to choose from 3 sequence types:
```